roles/StackStorm.mongodb/defaults/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
 # MongoDB default version
-mongodb_version: "4.0"
+mongodb_version: "{% if ansible_facts.os_family == 'Debian' and ansible_facts.distribution_major_version == '20' %}4.4{% else %}4.0{% endif %}"
```

Comment:
To decide between 4.0 and 4.4 mongo version, wouldn't it be a better idea to use st2 version > 3.4 rather than only on Ubuntu 20?

Reply:
@rushskills Stackstorm 3.5 still uses mongo 4.0 for all OSes EXCEPT for Ubuntu 30